server/src/routes/api/v2/admin/tokens.js

```diff
@@ -25,14 +25,16 @@ router.post('/mint', auth.required, async (req, res) => {
     return res.status(400).send({ error: 'The user is not a community admin' })
   }
   const { tokenAddress, networkType, amount, toAddress, correlationId } = req.body
-
   if (networkType !== 'fuse') {
     return res.status(400).send({ error: 'Supported only on Fuse Network' })
   }
-
-  const amountInWei = toWei(amount.toString())
-  const job = await agenda.now('mint', { tokenAddress, bridgeType: 'home', from: accountAddress, amount: amountInWei, toAddress, correlationId })
-  return res.json({ job: job.attrs })
+  try {
+    const amountInWei = toWei(amount.toString())
+    const job = await agenda.now('mint', { tokenAddress, bridgeType: 'home', from: accountAddress, amount: amountInWei, toAddress, correlationId })
```

Comment:
not sure the error inside the job is throw here

Reply:
Couldn't find any try/catch in the jobs except for a throw inside agenda.now definition...

Reply:
Anyway, failed jobs will have the fail error inside the object and can be retrieved by get-job endpoint.
This is more if something crashes in the middle.